### PR TITLE
Rec7 docker

### PR DESCRIPTION
**Pull request by tlee0818**
added recitation 7 information.

Link:
[slides](https://docs.google.com/presentation/d/1wjGBK0A09iW10XLUI9fqEThVysF3y3BULmeEMitKjM0/edit#slide=id.g21dccbf7d57_0_64)
[quiz](https://www.gradescope.com/courses/488839/assignments/2747781)

### CODE REVIEW

File: docs/recitations/reci7-docker.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "start_time": "HH:MM",
+    "end_time": "HH:MM",
+    "ta": ["taName1", "taName2"]
+}
+```
+
+ 
+## Activity
+
+1. Implement the `section_info` endpoint according to specifications. Test it by running the app locally.
```
Comment (anudaweerasinghe) [inline docs/recitations/reci7-docker.md:38]: Give them a little more guidance here. What file should the change? What's already in the file, what needs to change? Adding comments to `main.py` will also help.

File: docs/recitations/reci7-docker.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+2. Implement `./Dockerfile`. You can use the slides and [this link](https://docs.docker.com/engine/reference/builder/) as resources.
```
Comment (anudaweerasinghe) [inline docs/recitations/reci7-docker.md:39]: We probably need to give more guidance here... Can we at least include the [comments from the example dockerfile?[(https://github.com/CMU-313/s23-microservice-for-recitation/blob/main/Dockerfile) And then mention something like "as in the TA demo" in the handout?

File: docs/recitations/reci7-docker.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+3. Create the docker image using the command below, and check the image has been created.
+```terminal
+docker build -t YOUR_IMAGE_NAME  .
+```
+4. Implement `./docker-compose.yml`. You can use the slides for reference.
+5. Create a container using the docker image using the command below.
+```terminal
+docker-compose up -d 
+```
+6. Check the your container is running correctly by locally invoking the endpoint.
```
Comment (anudaweerasinghe) [inline docs/recitations/reci7-docker.md:49]: Some people may not know what "locally invoking the endpoint" means. Better to have a URL that they should try visiting

**Review by anudaweerasinghe — CHANGES_REQUESTED** [review]

**Comment by anudaweerasinghe** [comment]
@tlee0818 also make sure to edit `docs/_data/schedule.yaml` so the handout, slides and quiz show up on the home screen

File: docs/recitations/reci7-docker.md (hunk 2 above)
Comment (tlee0818) [reply docs/recitations/reci7-docker.md:39]: i give them link to the slides.

**Comment by tlee0818** [comment]
@anudaweerasinghe made changes accordingly. feel free to merge if it seems good!

**Review by anudaweerasinghe — APPROVED** [review]

**Pull request merged**